<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>samplv1 - An old-school polyphonic sampler

  Copyright (C) 2012-2015, rncbc aka Rui Nuno Capela. All rights reserved.

  This program is free software; you can redistribute it and/or
  modify it under the terms of the GNU General Public License
  as published by the Free Software Foundation; either version 2
  of the License, or (at your option) any later version.

  This program is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.

  You should have received a copy of the GNU General Public License along
  with this program; if not, write to the Free Software Foundation, Inc.,
  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>samplv1widget_config</class>
 <widget class="QDialog" name="samplv1widget_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure - samplv1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="samplv1.qrc">:/images/samplv1.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="TabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ControlsTab">
      <attribute name="title">
       <string>&amp;Controllers</string>
      </attribute>
      <attribute name="toolTip">
       <string>Controllers</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QToolButton" name="ControlsAddItemToolButton">
           <property name="toolTip">
            <string>Add controller</string>
           </property>
           <property name="text">
            <string>&amp;Add Controller</string>
           </property>
           <property name="icon">
            <iconset resource="samplv1.qrc">:/images/samplv1_control.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="ControlsEditToolButton">
           <property name="toolTip">
            <string>Edit controller</string>
           </property>
           <property name="text">
            <string>&amp;Edit</string>
           </property>
           <property name="icon">
            <iconset resource="samplv1.qrc">:/images/presetEdit.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="ControlsDeleteToolButton">
           <property name="toolTip">
            <string>Delete controller</string>
           </property>
           <property name="text">
            <string>&amp;Delete</string>
           </property>
           <property name="icon">
            <iconset resource="samplv1.qrc">:/images/presetDelete.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="samplv1widget_controls" name="ControlsTreeWidget">
         <property name="toolTip">
          <string>Controllers</string>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">Channel</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Parameter</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Subject</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QCheckBox" name="ControlsEnabledCheckBox" >
           <property name="toolTip" >
            <string>Whether MIDI controllers will be enabled</string>
           </property>
           <property name="text" >
            <string>E&amp;nabled</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ProgramsTab">
      <attribute name="title">
       <string>&amp;Programs</string>
      </attribute>
      <attribute name="toolTip">
       <string>Programs</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QToolButton" name="ProgramsAddBankToolButton">
           <property name="toolTip">
            <string>Add bank</string>
           </property>
           <property name="text">
            <string>Add &amp;Bank</string>
           </property>
           <property name="icon">
            <iconset resource="samplv1.qrc">:/images/presetBank.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="ProgramsAddItemToolButton">
           <property name="toolTip">
            <string>Add program</string>
           </property>
           <property name="text">
            <string>&amp;Add Program</string>
           </property>
           <property name="icon">
            <iconset resource="samplv1.qrc">:/images/samplv1_preset.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="ProgramsEditToolButton">
           <property name="toolTip">
            <string>Edit bank/program</string>
           </property>
           <property name="text">
            <string>&amp;Edit</string>
           </property>
           <property name="icon">
            <iconset resource="samplv1.qrc">:/images/presetEdit.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="ProgramsDeleteToolButton">
           <property name="toolTip">
            <string>Delete bank/program</string>
           </property>
           <property name="text">
            <string>&amp;Delete</string>
           </property>
           <property name="icon">
            <iconset resource="samplv1.qrc">:/images/presetDelete.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="samplv1widget_programs" name="ProgramsTreeWidget">
         <property name="toolTip">
          <string>Programs</string>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">Bank/Program</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Preset</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QCheckBox" name="ProgramsEnabledCheckBox" >
           <property name="toolTip" >
            <string>Whether MIDI bank/programs will be enabled</string>
           </property>
           <property name="text" >
            <string>E&amp;nabled</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="ProgramsPreviewCheckBox" >
           <property name="toolTip" >
            <string>Whether to preview the current selected program</string>
           </property>
           <property name="text" >
            <string>&amp;Preview current program</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="OptionsTab">
      <attribute name="title">
       <string>&amp;Options</string>
      </attribute>
      <attribute name="toolTip">
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="UseNativeDialogsCheckBox">
         <property name="toolTip">
          <string>Whether to use desktop environment's native dialogs where applicable</string>
         </property>
         <property name="text">
          <string>Use desktop environment &amp;native dialogs</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="KnobDialModeTextLabel">
         <property name="text">
          <string>&amp;Knob dial mode:</string>
         </property>
         <property name="buddy">
          <cstring>KnobDialModeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="KnobDialModeComboBox">
         <property name="toolTip">
          <string>Knob dial mode</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>(default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Angular</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
	   <item row="2" column="0">
        <widget class="QLabel" name="CustomStyleThemeTextLabel">
         <property name="text">
          <string>Custom &amp;style theme:</string>
         </property>
         <property name="buddy">
          <cstring>CustomStyleThemeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="CustomStyleThemeComboBox">
         <property name="toolTip">
          <string>Custom widget style theme</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>(default)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" colspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="8" margin="8"/>
 <customwidgets>
  <customwidget>
   <class>samplv1widget_controls</class>
   <extends>QTreeWidget</extends>
   <header>samplv1widget_controls.h</header>
  </customwidget>
  <customwidget>
   <class>samplv1widget_programs</class>
   <extends>QTreeWidget</extends>
   <header>samplv1widget_programs.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ControlsAddItemToolButton</tabstop>
  <tabstop>ControlsEditToolButton</tabstop>
  <tabstop>ControlsDeleteToolButton</tabstop>
  <tabstop>ControlsTreeWidget</tabstop>
  <tabstop>ControlsEnabledCheckBox</tabstop>
  <tabstop>ProgramsAddBankToolButton</tabstop>
  <tabstop>ProgramsAddItemToolButton</tabstop>
  <tabstop>ProgramsEditToolButton</tabstop>
  <tabstop>ProgramsDeleteToolButton</tabstop>
  <tabstop>ProgramsTreeWidget</tabstop>
  <tabstop>ProgramsEnabledCheckBox</tabstop>
  <tabstop>ProgramsPreviewCheckBox</tabstop>
  <tabstop>UseNativeDialogsCheckBox</tabstop>
  <tabstop>KnobDialModeComboBox</tabstop>
  <tabstop>CustomStyleThemeComboBox</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="samplv1.qrc"/>
 </resources>
 <connections/>
</ui>
